packages/renderer/src/lib/image/PullImage.svelte

```diff
@@ -128,13 +121,6 @@ function validateImageName(event): void {
       </span>
       Manage registries
     </button>
-
-    <button on:click="{() => toggleAddARegistry()}" class="pf-c-button pf-m-primary" type="button">
```

Comment:
I think here we should bring the new registry option. As we can still see that modal from the pull image page

Reply:
Ok there is manage registries